src/serac/physics/utilities/weak_form/boundary_integral.hpp

```diff
@@ -0,0 +1,633 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/**
+ * @file integral.hpp
+ *
+ * @brief This file contains the Integral core of the weak form
+ */
+#pragma once
+
+#include "mfem.hpp"
+#include "mfem/linalg/dtensor.hpp"
+
+#include "serac/physics/utilities/variational_form/tensor.hpp"
+#include "serac/physics/utilities/variational_form/quadrature.hpp"
+#include "serac/physics/utilities/variational_form/finite_element.hpp"
+#include "serac/physics/utilities/variational_form/tuple_arithmetic.hpp"
```

Comment:
More variational_form directories

Reply:
oops, this file wasn't supposed to be added!